### ISSUE TITLE
German Windows keyboard problem

### ISSUE THREAD

**Issue by Wikunia** [ohnnyj/brackets-numbertabs#4]
Hi,

first thanks for this extension! I have a problem since v0.0.4 because I can't write `{` anymore inside Brackets. On a German keyboard the combination to write `{` is `AltGr+7` which is the same as 
`Ctrl-Alt-7`. Maybe you can use `Ctrl` for the active pane and `Alt` for the inactive one? 
Hope there is no problem on other keyboards!

Have a nice weekend!

**Comment by asiertejada** [comment]
Just FYI, the fix for this issue creates another problem for Mac users with Spanish keyboards as @ or # are written by typing Alt+2 and Alt+3 respectively, thus rendering Brackets unusable if this extension is activated.

Reversing the changes made in [Fixes Issue #4](https://github.com/ohnnyj/brackets-numbertabs/commit/974c1702990036c2085c0fbbe19eedd9749bdac0) makes the deal.

**Comment by ohnnyj** [comment]
Thanks for the heads up.

It's going to be difficult to ensure compatibility across all keyboard types.

As a work around you can override keys in the keymap.json file (Debug -> Open User Key Map).

For example, to change the key binding for switching to tab number 1 in the active pane you can have:

```JavaScript
"overrides": {
  "Cmd-Shift-1":     "ohnnyj.numbertabs.active.1"
}
```

The other keys follow this pattern:

ohnny.numbertabs.(active|inactive).[1-9].

I know its probably not the best solution. You can also make a fork.
